types/yanzi/presenceMini/uplink.js

```diff
@@ -1,3 +1,12 @@
+function incrementValue(lastPulse, pulse) {
+  // Init state && Check for the case the counter reseted
+  if (lastPulse === undefined || lastPulse > pulse) {
+    lastPulse = pulse;
+  }
+  // Calculate increment
+  return pulse - lastPulse;
```

Comment:
make sure that this is always positive (for all instances of this)

Reply:
if (lastPulse === undefined || lastPulse > pulse) {

Should always catch sensor resets, which then equal 0